### ISSUE TITLE
Missing person class in predictions

### ISSUE THREAD

**Issue by iriyagupta** [prajjwal1/autonomous-object-detection#12]
Hi,

Thanks for releasing the code and the pretrained models. They work well, however, the person class is not getting predicted in any of the input images if the IDD model is used.
Do you have any idea why?

**Comment by prajjwal1** [comment]
Thanks for showing interest. I don't remember the class proportion. One possible reason that comes off the top, is maybe due to class imbalance. Could you check how many samples contain `person` class ?

**Comment by iriyagupta** [comment]
Hi, 

I don't think class imbalance is an issue, as the person class here is shown to have a lot of samples, in fact, more than the rest of the labels. For your reference, I am attaching an image showing the class distribution:

[https://idd.insaan.iiit.ac.in/static/img/detection/label_hierarchy.png](url)

**Comment by prajjwal1** [comment]
It's hard to debug from outside. Try to run inference on training or validation set. Feed images which have `person` present in them, and see if the model predicts it. I was getting a decent mAP, so I think it should predict `person` at some point if `person` class is dominant in this data distribution.

**Comment by prajjwal1** [comment]
Closing this, since it's not a bug and not related to implementation.
